workflow-job

```diff
@@ -492,7 +492,7 @@ flows["${sha1}"] = {
                 if (isAlive("james-project")) {
                     sh "docker rm -f james-project"
                 }
-                sh "docker run --hostname linagora.com -p '80:80' -p '25:25' -p '110:110' -p '143:143' -p '465:465' -p '587:587' -p '993:993' -p '8000:8000' --link cassandra:cassandra --link elasticsearch:elasticsearch -d --name james-project -t linagora/james-project"
+                sh "docker run --hostname linagora.com -p '80:80' -p '25:25' -p '110:110' -p '143:143' -p '465:465' -p '587:587' -p '993:993' -p '8000:8000' --link cassandra:cassandra --link elasticsearch:elasticsearch -d --name james-project -v /root/james/conf:/root/conf -t linagora/james-project"
```

Comment:
Can you put the host path in a variable (close to _continuousDeploymentDockerIp_)
by this way, we will not forget to extract this when integrating multiple deployments.